examples/custom-resources/externalname-services/transport-server/README.md

```diff
@@ -0,0 +1,101 @@
+# Support for Type ExternalName Services
+
+The Ingress Controller supports routing requests to services of the type [ExternalName](https://kubernetes.io/docs/concepts/services-networking/service/#externalname).
+
+An ExternalName service is defined by an external DNS name that is resolved into the IP addresses, typically external to the cluster. This enables to use the Ingress Controller to route requests to the destinations outside of the cluster.
+
+**Note:** This feature is only available in NGINX Plus.
+
+# Prerequisites
+
+
+For the illustration purpose we will run NGINX Ingress Controller (refered as NIC in the examples) with the ```-watch-namespace=nginx-ingress,default``` option. The option enables NIC to watch selected namespaces.
+
+Any application deployed in other namespaces will be treated as an external service.
+
+We will use the [tls-passthrough](../../tls-passthrough/README.md) application example as our backend app that will be responding to requests.
+
+# Example
+
+## 1. Deploy the tls-passthrough application
+
+1. Deploy the backend application as described in the [tls-passthrough example](../../tls-passthrough/README.md), and make sure it is working as described.
+
+## 2. Deploy external service to external namespace
+
+1. Navigate to the [external-name example](../../../custom-resources/externalname-services/README.md)
+
+2. Deploy external namespace (```external-ns```) and the backend application. Note that the namespace is not being watched by ```NIC```
+    ```
+    $ kubectl apply -f transport-server/secure-app-external.yaml
```

Comment:
I think we should remove the `$` from the start of these code blocks.
If a user is reading these docs and clicks the button to copy the full code block, they will also copy that `$` which will mess up the command

Reply:
@shaun-nx we included it to keep inline with other examples